{
  "uid" : "4b4757e66a1912dae1a509f688f20b0f",
  "children" : [ {
    "name" : "Test defects",
    "children" : [ {
      "name" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=43719): Max retries exceeded with url: /session/91742e4ead70df7aa6cbcad06f4f1bd2/url (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe18db6bc70>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "children" : [ {
        "name" : "test__registration",
        "uid" : "7edf29454996e51a",
        "parentUid" : "7ae78e53de203a86fae8f5d729f78233",
        "status" : "broken",
        "time" : {
          "start" : 1627892995669,
          "stop" : 1627892995669,
          "duration" : 0
        },
        "flaky" : false,
        "newFailed" : false,
        "parameters" : [ ]
      } ],
      "uid" : "7ae78e53de203a86fae8f5d729f78233"
    }, {
      "name" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=43719): Max retries exceeded with url: /session/91742e4ead70df7aa6cbcad06f4f1bd2/url (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe18d187070>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "children" : [ {
        "name" : "test__login",
        "uid" : "3b7e26627e5bd366",
        "parentUid" : "3eebee6d4af71b86f4fb55ab40f2ec87",
        "status" : "broken",
        "time" : {
          "start" : 1627892995899,
          "stop" : 1627892995899,
          "duration" : 0
        },
        "flaky" : false,
        "newFailed" : false,
        "parameters" : [ ]
      } ],
      "uid" : "3eebee6d4af71b86f4fb55ab40f2ec87"
    }, {
      "name" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=43719): Max retries exceeded with url: /session/91742e4ead70df7aa6cbcad06f4f1bd2/url (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe18d1ab5e0>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "children" : [ {
        "name" : "test__list_data",
        "uid" : "1a855daf79c1b31a",
        "parentUid" : "29c84105f7014ac0215d8ef8db215469",
        "status" : "broken",
        "time" : {
          "start" : 1627892996070,
          "stop" : 1627892996070,
          "duration" : 0
        },
        "flaky" : false,
        "newFailed" : false,
        "parameters" : [ ]
      } ],
      "uid" : "29c84105f7014ac0215d8ef8db215469"
    }, {
      "name" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=43719): Max retries exceeded with url: /session/91742e4ead70df7aa6cbcad06f4f1bd2/url (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe18d238760>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "children" : [ {
        "name" : "test__pagination",
        "uid" : "9782d02414b0c9c5",
        "parentUid" : "59396f33f8785421f55943a5cb7a86b5",
        "status" : "broken",
        "time" : {
          "start" : 1627892996255,
          "stop" : 1627892996255,
          "duration" : 0
        },
        "flaky" : false,
        "newFailed" : false,
        "parameters" : [ ]
      } ],
      "uid" : "59396f33f8785421f55943a5cb7a86b5"
    }, {
      "name" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=43719): Max retries exceeded with url: /session/91742e4ead70df7aa6cbcad06f4f1bd2/url (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe18d175220>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "children" : [ {
        "name" : "test__add_new_article",
        "uid" : "2d247d09d26b70c8",
        "parentUid" : "39a7b65cba12f5e977e5b6b1a0807de3",
        "status" : "broken",
        "time" : {
          "start" : 1627892996448,
          "stop" : 1627892996448,
          "duration" : 0
        },
        "flaky" : false,
        "newFailed" : false,
        "parameters" : [ ]
      } ],
      "uid" : "39a7b65cba12f5e977e5b6b1a0807de3"
    }, {
      "name" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=43719): Max retries exceeded with url: /session/91742e4ead70df7aa6cbcad06f4f1bd2/url (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe18d063be0>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "children" : [ {
        "name" : "test__import_data_from_file",
        "uid" : "2f17a8a9105caad7",
        "parentUid" : "94954d054211bb3fbb6a6bdba5294408",
        "status" : "broken",
        "time" : {
          "start" : 1627892996667,
          "stop" : 1627892996667,
          "duration" : 0
        },
        "flaky" : false,
        "newFailed" : false,
        "parameters" : [ ]
      } ],
      "uid" : "94954d054211bb3fbb6a6bdba5294408"
    }, {
      "name" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=43719): Max retries exceeded with url: /session/91742e4ead70df7aa6cbcad06f4f1bd2/url (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe18d0fa2e0>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "children" : [ {
        "name" : "test__modify_article",
        "uid" : "dc6f9e84ad36f756",
        "parentUid" : "d892b416220819caf863e5ff4c3f06e9",
        "status" : "broken",
        "time" : {
          "start" : 1627892996852,
          "stop" : 1627892996852,
          "duration" : 0
        },
        "flaky" : false,
        "newFailed" : false,
        "parameters" : [ ]
      } ],
      "uid" : "d892b416220819caf863e5ff4c3f06e9"
    }, {
      "name" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=43719): Max retries exceeded with url: /session/91742e4ead70df7aa6cbcad06f4f1bd2/url (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe18cfd50a0>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "children" : [ {
        "name" : "test__delete_article",
        "uid" : "a43b3afa1be26198",
        "parentUid" : "9ca17acdc211d59ba92f18311bcd2d52",
        "status" : "broken",
        "time" : {
          "start" : 1627892997039,
          "stop" : 1627892997039,
          "duration" : 0
        },
        "flaky" : false,
        "newFailed" : false,
        "parameters" : [ ]
      } ],
      "uid" : "9ca17acdc211d59ba92f18311bcd2d52"
    }, {
      "name" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=43719): Max retries exceeded with url: /session/91742e4ead70df7aa6cbcad06f4f1bd2/url (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe18d0195b0>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "children" : [ {
        "name" : "test__save_data_to_file",
        "uid" : "a759537b1f5edb68",
        "parentUid" : "f083927909f5767d7c87ecdf23a1ea21",
        "status" : "broken",
        "time" : {
          "start" : 1627892997235,
          "stop" : 1627892997235,
          "duration" : 0
        },
        "flaky" : false,
        "newFailed" : false,
        "parameters" : [ ]
      } ],
      "uid" : "f083927909f5767d7c87ecdf23a1ea21"
    }, {
      "name" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=43719): Max retries exceeded with url: /session/91742e4ead70df7aa6cbcad06f4f1bd2/url (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe18d2350a0>: Failed to establish a new connection: [Errno 111] Connection refused'))",
      "children" : [ {
        "name" : "test__logout",
        "uid" : "1761e7b3e983844c",
        "parentUid" : "a18f4adb82d555f877834e2f872eecaa",
        "status" : "broken",
        "time" : {
          "start" : 1627892997408,
          "stop" : 1627892997408,
          "duration" : 0
        },
        "flaky" : false,
        "newFailed" : false,
        "parameters" : [ ]
      } ],
      "uid" : "a18f4adb82d555f877834e2f872eecaa"
    } ],
    "uid" : "bdbf199525818fae7a8651db9eafe741"
  } ],
  "name" : "categories"
}